icons/circle-fading-arrow-up.json

```diff
@@ -0,0 +1,23 @@
+{
+  "$schema": "../icon.schema.json",
+  "contributors": [
+    "jordan808",
+    "jguddas",
+    "colebemis",
+    "ericfennis",
+    "mosch"
+  ],
+  "tags": [
+    "north",
+    "up",
+    "upgrade",
+    "improve",
+    "circle",
+    "button"
+  ],
+  "categories": [
+    "arrows",
+    "communication",
+    "shapes"
```

Comment:
These categories are not the correct ones. 
```suggestion
```

Reply:
`circle-fading-plus` also has `shapes` as a category, so I think that should be fine. `communication` I'm not 100% sold on either.